### PR TITLE
Remove upper case constants

### PR DESCRIPTION
**Pull request by lrhn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.15.2+16
+
+- Updated SDK version to 2.0.0-dev.17.0
```
Comment (munificent) [inline CHANGELOG.md]: Nit: Other entries below use `*` instead of `-` for bullets.

**Review by munificent — APPROVED** [review]
LGTM. Are you planning to publish this to pub too?

File: CHANGELOG.md (hunk 1 above)
Comment (lrhn) [reply CHANGELOG.md]: Ah, good catch. I'll have to check all the PRs for that.
(We apparently don't have a single consistent CHANGELOG format).

**Comment by lrhn** [comment]
I expect an owner of the package to do the push. I can do it if I have the necessary permissions, but I see no reason to believe that I do .... and after checking, I don't.

**Comment by munificent** [comment]
It's published now, thanks!

**Pull request merged**